packages/frontend/src/renderer/components/widgets/channels/CreateChannel/CreateChannel.tsx

```diff
@@ -65,7 +65,11 @@ const useStyles = makeStyles(theme => ({
 }))
 
 export const parseChannelName = (name = '') => {
-  return name.toLowerCase().replace(/ +/g, '-')
+  return name
+    .toLowerCase()
+    .trimStart()
```

Comment:
`trim` does the same as trimStart + trimEnd

Reply:
I've been trying to use it and it turned out it trims also the spaces between the words

Reply:
It doesn't. I use it in a different place and also check the docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim